servers/zts/src/main/java/com/yahoo/athenz/zts/cert/impl/DynamoDBCertRecordStoreConnection.java

```diff
@@ -54,17 +59,25 @@
     private static final String KEY_EXPIRY_TIME = "expiryTime";
     private static final String KEY_HOSTNAME = "hostName";
     private static final String KEY_TTL = "ttl";
+    public static final String INDEX_CURRENT_DATE = "currentDate-index";
```

Comment:
Note that we need to create a new index in prod and any other environment before pushing these changes

Reply:
In the conf/zts.properties for the athenz.zts.cert_dynamodb_table_name setting I have small section what type of table we need to create. can you also update that section to include the requirement for this index as well.